core/editable.js

```diff
@@ -2029,6 +2029,7 @@
 			while ( ( node = that.mergeCandidates.pop() ) )
 				node.mergeSiblings();
 
+			range.root.$.normalize && range.root.$.normalize();
```

Comment:
It would be safer to just normalize paragraph / block limit element which was modified instead of range root (which might be entire document), but not sure if we will be able to get such element correctly in all cases :thinking:

Reply:
I create a PR against the devoidfury fork replacing that line with:
`range.startPath().block.$.normalize()`

Also, I fixed the deleting issue in that PR https://github.com/devoidfury/ckeditor4/pull/1

Reply:
I created a PR against the devoidfury fork, replacing that line with
`range.startPath().block.$.normalize();`

Also, I added a possible fix to the deleting issue https://github.com/devoidfury/ckeditor4/pull/1.